.github/workflows/examine-wasm-bindings.yml

```diff
@@ -77,10 +77,25 @@ jobs:
         working-directory: wallet/bindings/wasm/
         run: yarn build
 
+      - name: Start private tangle
+        if: ${{ startsWith(matrix.os, 'ubuntu') }}
+        uses: './.github/actions/private-tangle/setup'
+
       - name: Run Yarn Test
+        if: ${{ startsWith(matrix.os, 'ubuntu') }}
         working-directory: wallet/bindings/wasm/
         run: yarn test
 
+      - name: Run Yarn Test
+        if: ${{ !startsWith(matrix.os, 'ubuntu') }}
+        working-directory: wallet/bindings/wasm/
+        # Only run test that doesn't require a node because we can't use the private tangle script on macos and windows
+        run: yarn test --t "generate mnemonic"
```

Comment:
Is there no way to make it work with attributes or something? Like in Rust ? This looks very error-prone as we'll 100% forget to add new tests here